test/container_test.py

```diff
@@ -92,8 +96,16 @@ def _run_container(
             # This is really only an issue for tests.
             sys.modules.pop(module_name)
 
+        env = os.environ.copy()
+        if is_checkpointing_function:
+            # Environment variable is set to allow restore from a checkpoint.
+            # Override server URL to reproduce restore behavior.
+            env["MODAL_FUNCTION_RESTORED"] = "1"
```

Comment:
Could be `_MODAL_FUNCTION_RESTORED` to indicate private.

Reply:
I'll drop this in favor of a state file mounted in the restore task.